services/terraformer/Dockerfile

```diff
@@ -50,6 +50,8 @@ COPY --from=build /go/services/terraformer/templates /bin/services/terraformer/t
 COPY --from=build /bin/grpc_health_probe /bin/grpc_health_probe
 COPY --from=build /bin/grpc_health_probe-Liveness /bin/grpc_health_probe-Liveness
 
+RUN apk add bash
```

Comment:
Why are we adding bash to the docker image ?

Reply:
Bash is a requirement for retry policy and is missing since #297

Reply:
Thanks for spotting this mistake.